manual/generator_actionmailer.md

```diff
@@ -0,0 +1,26 @@
+
+## ActionMailer
+
+`Tapioca::Compilers::Dsl::ActionMailer` generates RBI files for subclasses of `ActionMailer::Base`
+(see https://api.rubyonrails.org/classes/ActionMailer/Base.html).
+
+For example, with the following `ActionMailer` subclass:
+
+~~~rb
+class NotifierMailer < ActionMailer::Base
+  def notify_customer(customer_id)
+    # ...
+  end
+end
+~~~
+
+this generator will produce the RBI file `notifier_mailer.rbi` with the following content:
+
+~~~rbi
```

Comment:
Should it be `rb` everywhere?

Reply:
Since Github can properly syntax highlight RBI blocks and since it is semantically better to have `rbi` in there, I would like to leave them as `rbi` blocks.